build-custom-host-artifacts-windows.yml

```diff
@@ -0,0 +1,47 @@
+jobs:
+
+- job: BuildFunctionsCustomHostWindows
+  displayName: '[Windows] Build FunctionsCustomHost'
+
+  templateContext:
+    outputParentDirectory: $(Build.ArtifactStagingDirectory)
+    outputs:
+    - output: pipelineArtifact
+      displayName: Publish FunctionsCustomHost packages
+      path: $(Build.ArtifactStagingDirectory)/_functionsCustomHostPackagesWindows
+      artifact: _functionsCustomHostPackagesWindows
+
+  steps:
+  - task: UseDotNet@2
+    displayName: Install .NET SDK from global.json
+    inputs:
+      useGlobalJson: true
+
+  - task: DotnetCoreCLI@2
+    displayName: Dotnet Publish (win-x64)
+    inputs:
+      command: publish
+      publishWebProjects: false
+      zipAfterPublish: false
+      arguments: -c Release -r win-x64 -o $(Build.SourcesDirectory)/pkg_output/windows
+      workingDirectory: $(Build.SourcesDirectory)/host/src/FunctionsCustomHost
+
+  - task: DotnetCoreCLI@2
+    displayName: Dotnet Publish (win-arm64)
+    inputs:
+      command: publish
+      publishWebProjects: false
+      zipAfterPublish: false
+      arguments: -c Release -r win-arm64 -o $(Build.SourcesDirectory)/pkg_output/windows
+      workingDirectory: $(Build.SourcesDirectory)/host/src/FunctionsCustomHost
+
+  - task: CopyFiles@2
+    displayName: Copy files
+    inputs:
+      SourceFolder: $(Build.SourcesDirectory)/pkg_output/windows
+      # Publish output will include many other files. We only need func.exe, pdb & nethost.dll
+      Contents: |
+        func.exe
+        func.pdb
+        nethost.dll
+      TargetFolder: $(Build.ArtifactStagingDirectory)/_functionsCustomHostPackagesWindows
```

Comment:
Don't we need "PublishPipelineArtifact" task to publish the content  produced from the dotnet publish tasks above ?

Reply:
added let me know if it is ok